watertap/unit_models/zero_order/low_pressure_pump_zo.py

```diff
@@ -0,0 +1,70 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a low pressure pump unit
+"""
+
+from pyomo.environ import Constraint, Reference, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_pt, pump_electricity, ZeroOrderBaseData
+from idaes.core.util.constants import Constants
+
+# Some more information about this module
+__author__ = "Chenyu Wang"
+
+
+@declare_process_block_class("PumpElectricityZO")
+class PumpElectricityZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a low pressure pump unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "low_pressure_pump"
```

Comment:
Let's change tech_type to "pump_electricity" too

Reply:
Done